src/storer.ts

```diff
@@ -0,0 +1,45 @@
+import {WebPage} from "./web-page";
+/**
+ * Created by lukas on 3/27/17.
+ */
+export class Storer {
+
+    static azure = require('azure');
+    static crypto = require('crypto');
+
+    static blobService = Storer.azure.createBlobService('wetstorage',
+        'x7P3BHTnMpY+JB2dAHnzkaHVoz080IUSthfYDhTPFtYWNzIuwn70szGE+vIHn5S4BPad6gioTdlLafxkpszGHQ==');
+
+    static container = 'websites';
+
+    public static storeWebsite(webpage : WebPage) : void {
+        Storer.storeWebsiteBlob(webpage);
```

Comment:
I guess that storing is async. Could we have a callback here?
Will be required for later: we first have to store the website and only then add entries to the database. Otherwise someone could read from the DB and try to access the website before it was stored.